packages/core/examples/nonVolumeActorAPI/index.ts

```diff
@@ -0,0 +1,225 @@
+import {
+  RenderingEngine,
+  Types,
+  Enums,
+  getRenderingEngine,
+  CONSTANTS,
+} from '@cornerstonejs/core';
+import {
+  initDemo,
+  setTitleAndDescription,
+  addButtonToToolbar,
+  addDropdownToToolbar,
+  camera as cameraHelpers,
+} from '../../../../utils/demo/helpers';
+
+import vtkActor from '@kitware/vtk.js/Rendering/Core/Actor';
+import vtkSphereSource from '@kitware/vtk.js/Filters/Sources/SphereSource';
+import vtkMapper from '@kitware/vtk.js/Rendering/Core/Mapper';
+
+// This is for debugging purposes
+console.warn(
+  'Click on index.ts to open source code for this example --------->'
+);
+
+const { ViewportType } = Enums;
+const { ORIENTATION } = CONSTANTS;
+
+const renderingEngineId = 'myRenderingEngine';
+const viewportId = 'POLYDATA_SAGITTAL';
+
+// ======== Set up page ======== //
+setTitleAndDescription(
+  'Non Volume Actor in Volume Viewport API',
+  'Demonstrates how to interact with a Volume viewport with non volume actors.'
+);
+
+const content = document.getElementById('content');
+const element = document.createElement('div');
+element.id = 'cornerstone-element';
+element.style.width = '500px';
+element.style.height = '500px';
+
+content.appendChild(element);
+// ============================= //
+
+addButtonToToolbar({
+  title: 'Apply Zoom',
+  onClick: () => {
+    // Get the rendering engine
+    const renderingEngine = getRenderingEngine(renderingEngineId);
+
+    // Get the stack viewport
+    const viewport = <Types.IVolumeViewport>(
+      renderingEngine.getViewport(viewportId)
+    );
+
+    // Get the current camera properties
+    const camera = viewport.getCamera();
+
+    const parallelScale = camera.parallelScale * 0.5;
+
+    viewport.setCamera({
+      parallelScale,
+    });
+    viewport.render();
+  },
+});
+
+addButtonToToolbar({
```

Comment:
I recommend just to remove these boilerplate codes for buttons on toolbar to show how minimal is to render a nonVolume actor with all tools enabled.